### PR TITLE
Enable VR for LBA1

### PR DESCRIPTION
**Pull request by xesf**
**Preview here:** https://pr-572.lba2remake.net

### CODE REVIEW

File: src/controls/vr/mappings.ts [hunk 1]
```diff
@@ -64,6 +67,28 @@ export function applyMappings(
...
     const { controlsState } = game;
     const { components } = motionController;
+
+    const listBehaviours = [
+        BehaviourMode.NORMAL,
+        BehaviourMode.ATHLETIC,
+        BehaviourMode.AGGRESSIVE,
+        BehaviourMode.DISCRETE,
+    ];
+    const hasProtoPack = game.getState().flags.quest[LBA2Items.PROTO_PACK] ||
```
Comment (agrande) [inline src/controls/vr/mappings.ts:77]: That's the same index for LBA1&2, right? Otherwise this would evaluate to true on some random item.

**Review by agrande — APPROVED** [review]
Just tried it, looks very promising!

**Comment by xesf** [comment]
True, forgot about that. There is another place with the same issue.
I will push the fixes tomorrow.

**Comment by xesf** [comment]
> Just tried it, looks very promising!

Yeah, not bad. I did like the prison scene.

**Comment by xesf** [comment]
I've pushed the changes.

We need to refactor this game specific stuff that is starting to be spread all over the game.

**Pull request merged**